src/swdmt/redistricting/Redistrictor.java

```diff
@@ -68,18 +72,30 @@ public Region region() {
         int numberOfAugmentedDistricts
                 = theRegion.numberOfVoters() % numDistricts;
         Iterator<Location> locit = theRegion.locations().iterator();
```

Comment:
Why is an explicit iterator used rather than using the enhanced-for (for-each) loop>